actions/run_action.go

```diff
@@ -82,6 +83,21 @@ func (run *RunAction) Verify(context *debos.DebosContext) error {
 		args := strings.Split(run.Script, " ")
 		run.scriptPath = debos.CleanPathAt(args[0], context.RecipeDir)
 		run.scriptArgs = args[1:]
+
+		/* Check the script exists on the filesystem (following symlinks) */
+		stat, err := os.Stat(run.scriptPath)
+		if err != nil {
+			return err
+		}
+
+		if stat.IsDir() {
```

Comment:
Rather then checking if it's a Dir this should proably check if it's a regular file